src/components/browser-viewport/browser-viewport.vue

```diff
@@ -1,16 +1,15 @@
 <template>
-  <iframe :src="siteUrl" width="100%" height="100%">
+  <iframe :src="url" width="100%" height="100%">
```

Comment:
Can the width and height here come from the CSS? or does it need to be attributes?